app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -206,78 +206,75 @@ const ContentListView: React.FC = () => {
   const fetch = React.useCallback(
     async (filter: IContentListFilter & Partial<IContentListAdvancedFilter>) => {
       try {
-        if (!isLoading) {
-          setIsLoading(true);
-
-          let workOrders: IPaged<IWorkOrderModel> = {
-            page: 0,
-            quantity: 0,
-            total: 0,
-            items: [],
-          };
-          if (filter.pendingTranscript) {
-            // Make a request for transcript work orders.
-            const startDate = moment().add(-3, 'day').startOf('day');
-            const endDate = moment().endOf('day');
-            const woFilter: IWorkOrderFilter = {
-              page: 1,
-              quantity: 1000,
-              createdStartOn: startDate.toISOString(),
-              createdEndOn: endDate.toISOString(),
-              workType: WorkOrderTypeName.Transcription,
-              status: [
-                WorkOrderStatusName.Submitted,
-                WorkOrderStatusName.InProgress,
-                WorkOrderStatusName.Completed,
-                WorkOrderStatusName.Failed,
-              ],
-            };
-
-            const response = await findWorkOrders(woFilter);
-            workOrders = response.data;
-            filter.contentIds = workOrders.items
-              .filter((wo) => !!wo.contentId)
-              .map((wo) => wo.contentId!);
-          }
+        setIsLoading(true);
```

Comment:
Instead of managing a separate state value you can `useApp` and check if there is an active request for content.

Reply:
I can do it in another PR.